libbpf-rs/src/skeleton.rs

```diff
@@ -340,6 +340,36 @@ pub trait SkelBuilder<'a> {
 }
 
 /// A trait for opened skeleton.
+///
+/// In addition to the methods defined in this trait, skeletons that implement this trait will also
+/// have bespoke implementations of a few additional methods to facilitate access to global
+/// variables of the BPF program. These methods will be named `bss()`, `data()`, and `rodata()`.
+/// Each corresponds to the variables stored in the BPF ELF program section of the same name.
+/// However if your BPF program lacks one of these sections the corresponding rust method will not
+/// be generated.
+///
+/// The type of the value returned by each of these methods will be specific to your BPF program.
+/// A common convention for BPF CO-RE programs is to define a single global variable in the BPF
```

Comment:
What does the CO-RE part here signify? Whether CO-RE is used or not the described behavior should be the same, right? So just drop that bit?

Reply:
Good point, I'll remove this.